src/libraries/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -114,13 +113,13 @@ public partial class Process : System.ComponentModel.Component
         protected void OnExited() { }
```

Comment:
NIT: For `GetProcessesByName` method overloads its allowing null for parameter `processName` but might be not intentional

Reply:
I'm not sure I follow?

Reply:
oh I was not able to comment on the row, its related to row 108

Reply:
Right, GetProcessesByName allows null for processName, in which case it just treats it like "".

Reply:
Got it, fixing now